src/geometry/frustum.rs

```diff
@@ -75,4 +79,46 @@ where
             corner_from(S::one(), S::one(), S::one()),
         ]
     }
+
+    fn compute_edges(&self) -> ArrayVec<[Vector3<S>; 6]> {
+        // To compute the edges, we need the points, so it's more efficient to implement
+        // intersector() directly and compute the points only once. We still provide this
```

Comment:
So, the dependency chain is corners -> edges -> normals. The default implementation would call:
* compute_corners(): Computes corners
* compute_edges(): Computes corners and edges
* compute_face_normals(): Computes corners, edges and face normals
so we'd compute corners three times and edges twice.

We could pass the corners to `compute_edges()`, and the edges to `compute_face_normals()`, would that be preferable? That forces an ordering between the calls, even when the previous results are not needed (like for AABBs).

Reply:
This may be a solution. Or, alternatively, how about getting rid of `compute_edges()` and `compute_face_normals()` as well as the default implementation of `intersector()` and let just everyone implementing `ConvexPolyhedron` do the edge/face normal computation inside `intersector()`?

Reply:
Sounds good.

Reply:
Done.